run_unittests.py

```diff
@@ -5058,21 +5062,24 @@ def test_cross_find_program(self):
         testdir = os.path.join(self.unit_test_dir, '11 cross prog')
         crossfile = tempfile.NamedTemporaryFile(mode='w')
         print(os.path.join(testdir, 'some_cross_tool.py'))
-        crossfile.write('''[binaries]
-c = '/usr/bin/cc'
-ar = '/usr/bin/ar'
-strip = '/usr/bin/ar'
-sometool.py = ['{0}']
-someothertool.py = '{0}'
-
-[properties]
-
-[host_machine]
-system = 'linux'
-cpu_family = 'arm'
-cpu = 'armv7' # Not sure if correct.
-endian = 'little'
-'''.format(os.path.join(testdir, 'some_cross_tool.py')))
+        crossfile.write(textwrap.dedent('''\
+            [binaries]
+            c = '/usr/bin/{1}'
+            ar = '/usr/bin/ar'
+            strip = '/usr/bin/ar'
+            sometool.py = ['{0}']
+            someothertool.py = '{0}'
+
+            [properties]
+
+            [host_machine]
+            system = 'linux'
+            cpu_family = 'arm'
+            cpu = 'armv7' # Not sure if correct.
+            endian = 'little'
+            ''').format(
+                os.path.join(testdir, 'some_cross_tool.py'),
```

Comment:
__[Flake8]__ [E126] continuation line over-indented for hanging indent ([link](https://sider.review/gh/repos/19784232/pulls/5820#issue-4458745))

<sub>posted by Sider</sub>